LocalSettings.php

```diff
@@ -3657,6 +3662,11 @@
 			'autoconfirmed',
 			'sysop'
 		],
+    		'+allpediawiki' => [
```

Comment:
Should be ordered alphabetically

Reply:
> Should be ordered alphabetically

It is. The only one above it is `default`, and we always sort `default` at the top.

Reply:
Should be moved down below ahinfoboxeswiki I think

Reply:
@Universal-Omega Oh. I see what you mean. Sorry, thought I had it alphabetized, but yeah I _do_ think `default` should be at the top.

Reply:
Done. Just waiting on CI to finish.